files/Trimble_Tekla/Sprint_2_1.ifc

```diff
@@ -0,0 +1,113 @@
+#44= IFCLOCALPLACEMENT($,#43);
+#45= IFCSITE('0jf8GwS$L3YP5fxsJ17CHX',#9,'Site name',$,$,#44,$,$,.ELEMENT.,(60,11,10),(24,48,23),21590.,$,$);
+#46= IFCCLASSIFICATION('Source',$,$,'Name',$,$,$);
+#47= IFCRELASSOCIATESCLASSIFICATION('3oXo6BC7j7iOeSRjyj0mp$',#9,$,$,(#37),#46);
+#48= IFCPROJECTEDCRS('EPSG:3879','ETRS89 / GK25FIN','EPSG:3879',$,$,$,$);
+#49= IFCMAPCONVERSION(#24,#48,5000.,1520.,6500.,1.1,2.1,1.,2.,3.);
+#50= IFCPERSON($,$,'Given name',$,$,$,$,$);
+#51= IFCDOCUMENTINFORMATION('Identification','IfcDocumentInformation Name',$,'Location',$,$,$,$,$,(#50),$,$,'application/pdf',$,$,$,$);
+#52= IFCRELASSOCIATESDOCUMENT('0AFSHgSfX93eLqmntxMMn7',#9,$,$,(#37),#51);
+#53= IFCPERSON($,$,'Given name',$,$,$,$,$);
+#54= IFCDATASETINFORMATION('Identification','IfcDatasetInformation Name',$,'Location',$,$,$,$,$,(#53),$,$,'text/xm',$,$,$,$,'http://geosciml.org/geosciml/2.0/xsd/geosciml.xsd');
+#55= IFCRELASSOCIATESDATASET('3oXrCTpC5BGeSRC8j6_h$P',#9,$,$,(#37),#54);
+#56= IFCPERSON($,$,'Given name',$,$,$,$,$);
+#57= IFCLIBRARYINFORMATION('Name','Version',#56,'2023-03-17T12:39:06','Location',$);
+#58= IFCRELASSOCIATESLIBRARY('0PEE3YJ3L8RQSPpdMLGCjR',#9,$,$,(#37),#57);
+#59= IFCTUNNEL('2$M74xvK5FOvGEnGoq1S65',#9,'Tunnel name',$,$,#44,$,$,.ELEMENT.,.NOTDEFINED.);
+#60= IFCTUNNELPART('20vTfIXYfAPevvWLwb8hFH',#9,'TunnelPart name',$,$,#44,$,$,.ELEMENT.,.NOTDEFINED.,.NOTDEFINED.);
+#61= IFCLOCALPLACEMENT(#44,#43);
+#62= IFCALIGNMENT('2Sxv5OSKnEWR02TrT$AMeC',#9,'Name',$,$,#61,$,$);
+#63= IFCLOCALPLACEMENT(#61,#43);
+#64= IFCCARTESIANPOINTLIST2D(((0.,0.),(30.,0.),(45.,-15.),(60.,0.),(45.,15.)),$);
+#65= IFCINDEXEDPOLYCURVE(#64,(IFCLINEINDEX((1,2)),IFCARCINDEX((2,3,4)),IFCARCINDEX((4,5,2))),.F.);
+#66= IFCGEOMETRICREPRESENTATIONSUBCONTEXT('Axis','Model',*,*,*,*,#24,$,.MODEL_VIEW.,$);
+#67= IFCSHAPEREPRESENTATION(#66,'Axis','Curve2D',(#65));
+#68= IFCPRODUCTDEFINITIONSHAPE($,$,(#67));
+#69= IFCALIGNMENTHORIZONTAL('2TWCXVWkn9VBazjeWBGrnd',#9,'Name',$,$,#63,#68);
+#70= IFCCARTESIANPOINTLIST3D(((0.,0.,0.),(30.,0.,-1.),(45.,-15.,-2.),(60.,0.,0.),(45.,15.,2.),(30.,0.,1.)),$);
+#71= IFCINDEXEDPOLYCURVE(#70,(IFCLINEINDEX((1,2)),IFCARCINDEX((2,3,4)),IFCARCINDEX((4,5,6))),.F.);
+#72= IFCSHAPEREPRESENTATION(#66,'Axis','Curve3D',(#71));
+#73= IFCPRODUCTDEFINITIONSHAPE($,$,(#72));
+#74= IFCALIGNMENTVERTICAL('0WMb_$AsrAcuy2SFgkyJSl',#9,'Name',$,$,#63,#73);
+#75= IFCCARTESIANPOINT((0.,0.));
+#76= IFCALIGNMENTHORIZONTALSEGMENT('StartTag','EndTag',#75,0.,0.,0.,30.,$,.LINE.);
+#77= IFCALIGNMENTSEGMENT('3qU5S3vIb5juWGExtWnbKJ',#9,'Name','Description',$,$,$,#76);
+#78= IFCCARTESIANPOINT((30.,0.));
+#79= IFCALIGNMENTHORIZONTALSEGMENT('StartTag','EndTag',#78,4.71238898038469,15.,15.,47.1238898038469,$,.CIRCULARARC.);
+#80= IFCALIGNMENTSEGMENT('3$NmPu6vr4QAdmyZ64YgBt',#9,'Name','Description',$,$,$,#79);
+#81= IFCCARTESIANPOINT((60.,0.));
+#82= IFCALIGNMENTHORIZONTALSEGMENT('StartTag','EndTag',#81,1.5707963267949,15.,15.,47.1238898038469,$,.CIRCULARARC.);

```

Comment:
There is no `CLOTHOID` horizontal element. The checker complains, however we can ignore this in the report, if wished.

Reply:
Please ignore this.